packages/amplify_analytics_pinpoint/android/src/test/kotlin/com/amazonaws/amplify/amplify_analytics_pinpoint/AmplifyAnalyticsPinpointPluginTest.kt

```diff
@@ -96,92 +93,92 @@ class AmplifyAnalyticsPinpointPluginTest {
     fun flushEvents_returnsSuccess() {
 
```

Comment:
Is it possible to add lints to enforce common new line strategies in our code as well?

Reply:
Great question - I'll do some digging

Reply:
I was able to set this in Swift but not in Kotlin. For whatever reason, they don't provide it as an option, but I dId enable "no multiple blank lines" at least.